conf/prod.routes

```diff
@@ -4,3 +4,5 @@
 ->         /template                                                   template.Routes
 
 GET        /admin/metrics                                              com.kenshoo.play.metrics.MetricsController.metrics
+
+GET         /customs/register-for-cds/accessibility-statement                      @uk.gov.hmrc.customs.rosmfrontend.controllers.ApplicationController.accessibilityStatement()
```

Comment:
/customs/register-for-cds/accessibility-statement  -> /manage-email-cds/accessibility-statement